src/app/shared/components/template/components/audio/audio.component.scss

```diff
@@ -42,7 +45,7 @@
   }
   .progress-block {
     .audio-range {
-      --bar-background-active: var(--ion-btn-control-bg);
+      --bar-background-active: #{$ion-btn-control-bg};
```

Comment:
nit(blocking) - just to double-check - do we need to interpolate these variables using `#{$ ... }` syntax? 
I don't remember doing this in the other components but might just be my mind blanking (I know you can just use variables directly like `$ion-btn-control-bg` , but do the requirements change if referencing var statements??

Reply:
We do in these cases where we're declaring a `--` css custom property, explanation here: https://sass-lang.com/documentation/style-rules/declarations#custom-properties